binarytree/__init__.py

```diff
@@ -649,6 +685,89 @@ def convert(bt):
     raise ValueError('Expecting a list or a node')
 
 
+def get_level(bt, level, show_values=False, show_nulls=False):
+    """Return the requested level of the binary tree, ordered from left to right.
+
+    If a node other than the root node is passed in, then this function
+    returns the requested level of the tree relative to the passed in node.
+
+    :param bt: The binary tree.
+    :type bt: binarytree.Node
+    :param level: The requested level to return.
+    :type level: int
+    :param show_values: whether to convert nodes to values before returning.
+    :type show_values: boolean
+    :param show_nulls: whether to show where empty nodes are.
+    :type show_nulls: boolean
+    :return: dictionary of form {0:[rootnode], 1:[1st, level, nodes]}
+    :rtype: dictionary
+    """
+    if not isinstance(bt, Node):
+        raise ValueError("Expecting a Node.")
```

Comment:
For the user's convenience, before raising an error I would check if the binarytree is in a list representation form. If it is try to convert to tree form first.

Reply:
So I can just say `bt = _prepare_tree(bt)`, correct?